app/src/main/java/com/vultisig/wallet/ui/models/keysign/JoinKeysignViewModel.kt

```
@@ -1462,40 +1474,48 @@ constructor(
+                        customMessagePayload ?: error("customMessagePayload unexpectedly null")
+                    messagesToSign = SigningHelper.getKeysignMessages(payload)
+                }
+
+                else ->
+                    throw KeysignMessagesException(
+                        "Both keysign payload and custom message payload are null"
+                    )
+            }
+        } catch (e: KeysignMessagesException) {
+            throw e
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)